apim-operator/deploy/sample/wso2_v1alpha1_security_cr.yaml

```diff
@@ -1,11 +1,35 @@
 apiVersion: wso2.com/v1alpha1
 kind: Security
 metadata:
-  name: example-security-test-oauth
-  namespace: wso2-system
+  name: oauthtest
+  namespace: <namespace>
```

Comment:
Shall we remove this namespace field? Otherwise, without editing this no body can deploy this...

Reply:
removed